front/src/applications/operationalStudies/components/Scenario/getTimetable.jsx

```diff
@@ -18,8 +18,9 @@ import i18n from 'i18next';
  * Recover the time table for all the trains
  */
 
-export default async function getTimetable(givenTimetableID) {
-  const timetableID = givenTimetableID || store.getState().osrdconf.timetableID;
+export default async function getTimetable(storedTimeTableID, givenTimetableID) {
+
+  const timetableID = givenTimetableID || storedTimeTableID;
```

Comment:
```suggestion
export default async function getTimetable(timetableID) {
```
This test should be done outside the function for more clarity (before the call). Also, the name of the function is not very clear, maybe fetchTimetable would be more appropriate (since it calls the API).

Reply:
yes